From ecfdb209afe2aafc378baab8c511f5df7b000270 Mon Sep 17 00:00:00 2001
From: Sedat Dilek <sedat.dilek@googlemail.com>
Date: Fri, 25 Feb 2011 20:48:14 +0000
Subject: UMS: Fix build against xserver 1.10-rc3

This issue was introduced due to last minute backout of RandR-1.4
in xserver 1.10-rc3.

Switch to "#ifdef RANDR_14_INTERFACE" as suggested by Keith Packard.
See also <http://lists.x.org/archives/xorg-devel/2011-February/019643.html>.

Note:
The ddx needs a rebuild as the X video driver ABI changed to version 10.0.

Reported-by: Alex Deucher <alexdeucher@gmail.com>
CC: Keith Packard <keithp@keithp.com>
Signed-off-by: Sedat Dilek <sedat.dilek@gmail.com>
---
diff --git a/src/radeon_output.c b/src/radeon_output.c
index 15cef06..ccde346 100644
--- a/src/radeon_output.c
+++ b/src/radeon_output.c
@@ -1622,7 +1622,7 @@ radeon_set_mode_for_property(xf86OutputPtr output)
 	xf86CrtcPtr crtc = output->crtc;
 
 	if (crtc->enabled) {
-#if XORG_VERSION_CURRENT >= XORG_VERSION_NUMERIC(1,9,99,901,0)
+#ifdef RANDR_14_INTERFACE
 	    xf86CrtcSetRec crtc_set_rec;
 
 	    crtc_set_rec.flags = (XF86CrtcSetMode |
--
cgit v0.8.3-6-g21f6
